keel-core/src/main/kotlin/com/netflix/spinnaker/keel/veto/unhappy/UnhappyVeto.kt

```diff
@@ -52,8 +52,8 @@ class UnhappyVeto(
     val resourceId = resource.id
     val application = resource.application
 
-    // maxDiff count represents the number of times we're allowed to see a diff and take action to try and fix it.
-    if (diffFingerprintRepository.actionTakenCount(resourceId) <= maxDiffCount(resource)) {
+    // maxActionCountPerDiff represents the number of times we're allowed to see a diff and take action to try and fix it.
+    if (diffFingerprintRepository.actionTakenCount(resourceId) <= maxActionCountPerDiff(resource)) {
```

Comment:
Tangentially related. This was a misnomer, IMO, especially since we actually have a column in the `diff_fingerprint` table with the diff count (not to be confused with `count_actions_taken`, which is what this code is about).

Reply:
P.S. We should consider changing the corresponding spec property as well in a future PR.